.github/workflows/helm_charts_scalar.yml

```diff
@@ -102,6 +112,10 @@ jobs:
       - name: Set up chart-testing
         uses: helm/chart-testing-action@v2.0.1
 
+      - name: Add Helm repos
+        run: |
+          helm repo add scalar https://scalar-labs.github.io/helm-charts
+
```

Comment:
`chart testing` is designed to run `helm dependency` at runtime, so we don't run it specifically